### PR TITLE
Fix  'NoneType' object has no attribute 'get_text'

### PR DESCRIPTION
**Pull request by fernandog**
@miigotu 
`info_hash = item.find('info_hash').get_text(strip=True)nAttributeError: 'NoneType' object has no attribute 'get_text'n'``

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

**Comment by fernandog** [comment]
@Labrys even if I check "if info_hash ..." in the inline?

**Comment by labrys** [comment]
``` python
item.find('info_hash').get_text(strip=True) if item.find('info_hash') else None
```

@fernandog yes because when it doesn't have an info_hash you are setting it to None (`else None`)

File: sickbeard/providers/extratorrent.py [hunk 1]
```diff
@@ -84,8 +84,8 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
                                 download_url = enclosure['url'] if enclosure else item.find('link').next.strip()
                                 download_url = re.sub(r'(.*)/torrent/(.*).html', r'\1/download/\2.torrent', download_url)
                             else:
-                                info_hash = item.find('info_hash').get_text(strip=True)
-                                download_url = "magnet:?xt=urn:btih:" + info_hash + "&dn=" + title + self._custom_trackers
+                                info_hash = item.find('info_hash').get_text(strip=True) if item.find('info_hash') else None
+                                download_url = "magnet:?xt=urn:btih:" + info_hash + "&dn=" + title + self._custom_trackers if info_hash else None
```
Comment (miigotu) [inline sickbeard/providers/extratorrent.py]: if info_hash and title else None

**Pull request merged**